concordium-std/tests/derive-deserial-with-state/success-ident-param.rs

```diff
@@ -0,0 +1,9 @@
+use concordium_std::{DeserialWithState, HasStateApi, ParseResult, Read, StateMap};
```

Comment:
Please document the intent of this test as a module doc comment, i.e., `//!`.

Reply:
Added